docs/hub-deployment-guide/upgrade-cluster/aws.md

```diff
@@ -0,0 +1,208 @@
+(upgrade-cluster:aws)=
```

Comment:
I would suggest that the upgrade-cluster guide live within the `howto` or `sre-guide` subdirectories